src/App.tsx

```
@@ -8,53 +8,182 @@ import { PostsList } from './components/PostsList';
+import { getUsers } from './Api/Users';
+import { getPosts } from './Api/Posts';
+import { useEffect, useState } from 'react';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import { createComments, deleteComment, getComments } from './Api/Comment';
+import { Comment } from './types/Comment';
 
-export const App = () => (
-  <main className="section">
-    <div className="container">
-      <div className="tile is-ancestor">
-        <div className="tile is-parent">
-          <div className="tile is-child box is-success">
-            <div className="block">
-              <UserSelector />
-            </div>
+export const App = () => {
+  const [users, setUsers] = useState<User[]>([]);
 
-            <div className="block" data-cy="MainContent">
-              <p data-cy="NoSelectedUser">No user selected</p>
+  const [isLoading, setIsLoading] = useState(false);
+  const [isLoadingComments, setIsLoadingComments] = useState(false);
 
-              <Loader />
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [selectedPost, setSelectedPost] = useState<Post>();
 
-              <div
-                className="notification is-danger"
-                data-cy="PostsLoadingError"
-              >
-                Something went wrong!
-              </div>
+  const [error, setError] = useState(false);
+  const [errorComments, setErrorComments] = useState(false);
+
+  const [isUserSelected, setIsUserSelected] = useState(false);
+  const [selectedUserId, setSelectedUserId] = useState<number | null>(null);
+
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+
+  const [comments, setComments] = useState<Comment[]>([]);
+
+  async function fetchCreateComment({
+    name,
+    email,
+    body,
+    postId,
+  }: Omit<Comment, 'id'>) {
+    try {
+      const result = await createComments({ name, email, body, postId });
+
+      setComments(currentComments => [...currentComments, result]);
+    } catch {}
```

Empty catch block here swallows errors silently. The code should at least log the error and/or set an error state so failures are visible (and tests can assert error states). This directly violates the previously noted requirement: "Also fix empty catch blocks in `App.tsx` to at least log or set error state".

Suggested fix: change to `} catch (err) { console.error(err); /* optionally set an error flag to show UI feedback */ }` so failures are visible and actionable.